datastream-kafka/src/main/java/com/linkedin/datastream/kafka/KafkaTestUtils.java

```diff
@@ -102,6 +104,24 @@ public static void waitForTopicCreation(ZkUtils zkUtils, String topic, String br
     throw new IllegalStateException("Topic was not ready within the timeout");
   }
 
+  /**
+   * Check if the given topic exists yet or not.
+   *
+   * @param adminClient AdminClient instance to check if topics exists
+   * @param topic Topic name
+   */
+  public static boolean topicExists(AdminClient adminClient, String topic) {
```

Comment:
I tried to import "topicExists" method from BaseKafkaZkTest class here but I kept getting "package com.linkedin.datastream.testutil does not exist" error so I copied the method here. Its probably some classpath issue but I am not sure how to resolve it in this repo.

Reply:
Yes, correct.

Reply:
If it's a circular dep issue, then could we reference this function from BaseKafkaZkTest class's call to avoid duplicate code?